src/Paket.Core/Versioning/SemVer.fs

```diff
@@ -125,40 +160,83 @@ type SemVerInfo =
     
     member x.AsString
         with get() = x.ToString()
+        
+    member x.Equals(y) =
+        x.Major = y.Major && x.Minor = y.Minor && x.Patch = y.Patch && x.Build = y.Build && x.PreRelease = y.PreRelease
```

Comment:
comparing BuildMetaData is breaking semver2 compliance, and was not used anywhere

Reply:
good catch, that never should have been there